### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by milen-sage**
Added more detailed developers' instructions.

@linglp could you take a look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,8 +150,36 @@ Most Google sheet functionality could be authenticated with service account. How
...
+7. Keep commit messages descriptive but less than a page long, see best practices
+8. Push code to develop-<feature/fix-name> in upstream repo
+9. Branch out off develop-<feature/fix-name> if needed to work on multiple features associated with the same code base
+10. After feature work is complete and before creating a PR to the develop branch in upstream
+    a. ensure that code runs locally
+    b. test for logical correctness locally
+    c. wait for git workflow to complete (e.g. tests are run) on github
+11. Create a PR from develop-<feature/fix-name> into the develop branch of the upstream repo
+12. Request a code review on the PR
+13. Once code is approved merge in the develop branch
```
Comment (linglp) [inline README.md:176]: Once code is approved, resolve conflicts and merge in the develop branch?

**Review by linglp — APPROVED** [review]
@milen-sage this looks good. I am also wondering if we should mention about linking GitHub issues to PR. Should we add that as an additional step too?

**Comment by milen-sage** [comment]
Yes, let's do that.

File: README.md (hunk 1 above)
Comment (milen-sage) [reply README.md:176]: Feel free to edit directly in the PR.

**Pull request merged**